projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
 use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\get_purchases; // phpcs:ignore MediaWiki.Classes.UnusedUseStatement.UnusedUse
```

Comment:
This function doesn't seem to be defined in the `Automattic\Jetpack\Connection\` namespace, but located in the class `Jetpack_Core_API_Site_Endpoint`.
Should we remove this import, or I'm missing something?

Reply:
Besides that, everything tests well and looks good.

Reply:
I removed the import after running the test suites and manually ensuring that the `jetpack/v4/site/purchases` endpoint still works as expected. 

However, I can't say for sure why it was added in the first place.

cc @fgiannar

Reply:
Thanks, @rcanepa!

I didn't add it myself, only the phpcs ignore to fix some linter warnings after adding `use Automattic\Jetpack\Connection\Rest_Authentication;`

`get_purchases` is defined in [Jetpack_Core_API_Site_Endpoint](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-site-endpoints.php#L67) so we should be good here :)